bugsnag-android-unity/src/main/java/com/bugsnag/android/unity/UnityClient.java

```diff
@@ -10,15 +10,19 @@
 
 public class UnityClient {
 
-    static Client client;
+    static volatile Client client;
+    private static Object lock = new Object();
 
     public static void init(Context androidContext, String apiKey, boolean trackSessions) {
-        if (client != null) {
-            client.disableExceptionHandler();
+        if (client == null) {
```

Comment:
This check is already carried out in the `synchronized` block so could be removed.

Reply:
This avoids taking the 'expensive' lock when we don't need to